app/views/volunteers/region_admin.html.erb

```diff
@@ -127,3 +129,5 @@
     <% end %>
   </div>
 </div>
+
+<%= javascript_include_tag "datepicker_footer" %>
```

Comment:
Can you add a line break after this? New lines are good at ends of files to prevent churn in git commits. https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file